### DISCUSSION TITLE
How to filter Office 365 Users with ComboBox (replicate "people picker")

### DISCUSSION THREAD

**Discussion by TerranIncognito** [microsoft/powercat-creator-kit#145] [Q&A] [answered]
In standard canvas apps, you can insert a ComboBox and perform a lookup on Office 365 Users using something like the following in the "Items" property:

Office365Users.SearchUser({searchTerm:ComboBox1.SearchText})

However, I am not seeing the "SearchText" property in the controls associated with the Creator Kit. 

What's the best way to do this? I'm trying to replicate "people picker" functionality in an older canvas app.

**Comment by denise-msft** [comment]
@TerranIncognito which component are you trying to use? There are specific input properties needed to render a collection in the Creator Kit controls. Each might be slightly different, so you need to rely on the documentation. 

For example, if you're trying to render a collection of users in the new Facepile control, use the sample in the docs: https://learn.microsoft.com/en-us/power-platform/guidance/creator-kit/facepile#office-365-users-connector

**Comment by denise-msft** [comment, marked as answer]
@TerranIncognito please check out the new [PeoplePicker](https://learn.microsoft.com/en-us/power-platform/guidance/creator-kit/peoplepicker) control

**Reply by colvinj** [reply]
This looks great! I'll start testing right away...many thanks!

**Reply by colvinj** [reply]
@denisem-msft I'm following the documentation, but am unable to get the items property to set to Office365Users. Even when I try to set the datasource in the Controls pane, the component will not accept any value.

**Reply by colvinj** [reply]
@denisem-msft please see attached screenshots.
![Suggestions_Items](https://user-images.githubusercontent.com/116035876/200040901-0cf3830a-ad73-422e-ac66-a44361449ce9.png)
![Items](https://user-images.githubusercontent.com/116035876/200040902-b52bcd38-fa73-4bc9-8920-70bc1381495f.png)
